frontend/src/components/pages/AdminDashboard/DeleteConfirmation.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from "@chakra-ui/react";
+import React from "react";
+
+export type DeleteConfirmationProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  onDelete: () => Promise<void>;
+  reviewName: string;
+};
+
+const DeleteConfirmation = ({
```

Comment:
Can we call this DeleteConfirmationModal instead?

Reply:
+1